news/2022-08-25-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,139 @@
+
+We will also be using terminology coined by the osu!taiko community to refer to how patterns are placed. **d** refers to don (red) notes, and **k** refers to kat (blue) notes. Repeated notes of the same colour are referred to as *monos* (short for monocolour).
+
+If you would like to know more about the changes posted, or for a more detailed breakdown on the some of the in-development changes, head on over to the [osu!taiko rework document](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit). This is a working document, and will periodically be updated with the latest developments from any developers who request their changes to be added to it, via the osu!taiko pp committee.
+
+## An insight into the base of the new rework
+
+The previous rework effort aimed to the tackle all three skills at once. This not only made it difficult to collect meaningful and targeted feedback, but halted continuous development efforts. As such, we opted for a more modular approach, by introducing the new changes incrementally, in hopes of being able to fix reported issues one stage at a time.
+
+Moving forward, osu!taiko star rating will be receiving continuous updates, potentially placing star rating and performance points in a state of flux, particularly over the coming months. During this time, we have been and will continue to work on other difficulty components (both small and large-scale) in preparation for future releases. This is generally desirable as we would want difficulty to accurately reflect the latest trends, but we understand and apologise if this may bring some unintended complications or frustrations, especially in the short term.
+
+In the star rating part of this rework, we have re-evaluated and essentially rewritten the stamina and colour skills, introducing new concepts and supporting infrastructure to the complex calculations that you see as star rating.
+
+As for the performance points part, not only have mod multipliers been reconsidered, but you will also see a new way in which accuracy will affect your performance points. However, it is important to note that these changes are very experimental, as performance was not the priority of this rework.
+
+Without further ado, let's get into the nitty-gritty of these changes.
+
+## Star rating changes
+
+### Readdressing stamina
+
+#### Issues within the old system
+
+The old system assumed a 2-finger kddk full alternating playstyle (i.e. one in which each subsequent hit is always hit by the other hand regardless of colour). This was implemented as two skills — one for the left hand, and one for the right hand. Instead of using note duration for the split skills, it uses note-pair duration, which is defined as the duration between two notes that were pressed with the same hand.
+
+**Never assume a specific playstyle or arbritrarily nerf patterns without a strong justification.**
```

Comment:
this paragraph comes way too much out of the blue with nothing connecting it to the previous paragraph

Reply:
Yeah don't really think we wanted to delete this during discussion, restored in 63880bfb15b1255ccc875c55f7bfeb794d590e70

Might wanna take a look at wordings etc though, as it might be removed temporarily for changes back then